grafana-plugin/src/pages/incidents/Incidents.tsx

```diff
@@ -521,10 +554,12 @@ class Incidents extends React.Component<IncidentsPageProps, IncidentsPageState>
   }
 
   setPollingInterval(filters: IncidentsFiltersType = this.state.filters, isOnMount = false) {
+    return;
```

Comment:
huh? `return` here?

Reply:
nice catch, I disabled polling locally due to performance optimisation